### DISCUSSION TITLE
Can you increase macOS video ram?

### DISCUSSION THREAD

**Discussion by YassineElkefi** [quickemu-project/quickemu#954] [Quickemu] [answered]
Is there any possibility to increase macOS ventura Display from 7mb to 128mb ? or even more ? i have Nvidia GEFORCE GTX 1650.

**Comment by lj3954** [comment, marked as answer]
No. There is no support for 3D graphics on macOS VMs. You would need to passthrough a supported GPU, likely only older AMD dGPUs or Intel iGPUs. Quickemu also does not currently have a simple method of GPU passthrough. For advanced features like that, you would likely be better off for now with a project like OSX-KVM. 

Overall, the macOS VMs are intended more for tasks like software development for Apple platforms, rather than for anything involving 3D graphics.

**Comment by P4rthPat3l** [comment]
Can we increase it to at least 128MB? I installed macOS, but it only shows 3MB of display memory, which isn't enough even for basic tasks.

**Reply by lj3954** [reply]
Quickemu allocates 256MB of graphics memory to macOS guests. The 3MB reported in the guest is erroneous. 3D graphics emulation is unsupported, which is why you may experience issues with more demanding graphical tasks.
